CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -30,7 +30,7 @@ Keadm adds some new sub-commands to improve the user experience, including conta
 
 - `beta join`: Installing edgecore as system service from docker image, no need to download from github release.
 
-- `beta reset`: Installing edgecore as system service from docker image, no need to download from github release.
+- `beta reset`: Automatically check the cloud side, the cloud service will be removed from Kubernetes and the side service will be cleared.
```

Comment:
Thanks for the fixing, and the reset cmd will also cleanup edgenode. So How about:
```
`beta reset`: Reset the node, clean up the resources installed on the node by `beta init` or `beta join`. It will automatically detect the type of node to clean up.
```
cc @gy95

Reply:
yes, this is better. @orangegzx please fix again :) thanks

Reply:
done